_site/resources/discounts/api/openapi.yml

```
@@ -19,22 +19,53 @@ components:
         lang:
           type: string
           description: language identifier
-          example: es_AR
+          example: 'es_AR'
         name:
           type: string
           description: Promotion name
-          example: 50% off in the second unit
+          example: '50% off in the second unit'
         description:
           type: string
           description: Describe the promotion to the consumer
-          example: Taking two pay 50 percent on the second unit
+          example: 'Taking two pay 50 percent on the second unit'
         disclaimer:
           type: string
           description: Describe the terms and conditions of the promotion
-          example: Deal only valid for Buenos Aires city
+          example: 'Deal only valid for Buenos Aires city'
       required:
         - lang
         - name
+    discountLineItem:
+      type: object
+      properties:
+        line_item:
+          type: string
+          description: Product to apply the promotion.
```

Normalicemos el uso del punto final. Creo que dejaría en todos lados sin punto, que pensas?  

```suggestion
          description: Product to apply the promotion
```